frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -129,6 +130,11 @@ private boolean isManagementReq(String[] segments) {
                 || endpointMap.containsKey(segments[1]);
     }
 
+    private boolean checkModelControl() {
+        ConfigManager configManager = ConfigManager.getInstance();
+        return (configManager.getModelControlMode().equals("explicit"));
+    }
+
```

Comment:
1. it is not efficient to compare string each time. 
2. better to rename as isModelControlled
3. move it to ConfigManager.
4. and also it is kind of redundancy getModelControlMode

Reply:
Noted will move the check into ConfigManager

Reply:
issue 1 is still not addressed.

Reply:
Issue 1 has been resolved as the string comparison only happens once and then is a true/false directly starting from 2nd time